scripts/openstack-quickstart-demosetup

```diff
@@ -787,6 +790,36 @@ if [ "x$with_gnocchi" = "xyes" ]; then
     #start_and_enable_service openstack-gnocchi-statsd
 fi
 
+#-----------------------------------------
+# setup Monasca
+#-----------------------------------------
+
+if [ "x$with_monasca" = "xyes" ]; then
+# monasca-log-agent
+    cp /etc/monasca-log-agent/agent.conf.sample /etc/monasca-log-agent/agent.conf
+    c="/etc/monasca-log-agent/agent.conf"
+    #TODO(sjamgade): change these the following two ips to $IP
+    sed -i "s:\(monasca_log_api_url =>\).*:\1 \"http\://$MONASCA_HOST\:8070/v3.0\":" $c
+    sed -i "s:\(keystone_api_url =>\).*:\1 \"http\://$IP\:5000/v3/\":" $c
+    sed -i "s:\(project_name =>\).*:\1 \"admin\":" $c
+    sed -i "s:\(username =>\).*:\1 \"admin\":" $c
+    sed -i "s:\(password =>\).*:\1 \"$pw\":" $c
+    sed -i "s:\(user_domain_name =>\).*:\1 \"Default\":" $c
+    sed -i "s:\(project_domain_name =>\).*:\1 \"Default\":" $c
+    sed -i "s:\(dimensions => \).*:\1 [ \"hostname\:monasca\" ]:" $c
+    start_and_enable_service  openstack-monasca-log-agent
+
+# monasca-log-agent
+    monasca-setup \
+    -u 'admin' \
+    -p '$pw' \
+    --skip_enable \
+    --user 'monasca-agent' \
+    --keystone_url "http://$IP:5000/v3/" \
```

Comment:
I think we have a variable that contains the keystone endpoint somewhere else already.. lets make use of that one